autobahn/wamp/cryptobox.py

```diff
@@ -231,7 +231,7 @@ def decode(self, is_originating, uri, encoded_payload):
             if encoded_payload.enc_serializer != u'json':
                 raise Exception("received encrypted payload, but don't know how to process serializer '{}'".format(encoded_payload.enc_serializer))
 
-            payload = _json_loads(payload_ser)
+            payload = _json_loads(payload_ser.decode('utf8'))
```

Comment:
when does that apply? cryptobox works for me

Reply:
I couldn't get it working at all (on py3) without that line -- the other side is encoding to `utf8`-bytes and JSON wants strings not bytes...